nih-sparc/sparc-app
# Contact form - cu-jjvezd

# Description
The purpose of this PR is to add contact forms to the portal. These two forms are for general inquiries and for reporting a bug. For general inquiries, it will send an email to SPARC. For bugs, it will create a wrike ticket.

## Tickets
https://app.clickup.com/t/jjvezd
https://www.wrike.com/open.htm?id=467362300

## Type of change


# How Has This Been Tested?

- In the header, click on "Contact Us'
- You should be taken to the contact us page
- Select `I couldn’t find the information in Help` from the dropdown
- The general form should be shown
- Clicking submit should notify you what fields are required
- Fill out the form and click "submit"
  - Checking "Subscribe to the SPARC Newsletter" should add that email to the Mailchimp list. Ask Leo for access to verify.
- You should now see a success message with your first name displayed
- Click on "Submit another inquiry"
- Select `I want to report an error or a technical issue` from the dropdown
- Clicking submit should notify you what fields are required
- Fill out the form and click "submit"
- You should now see a success message with your first name displayed
- Go to Wrike and verify that you ticket has been created, and the ticket's body has the correct info you added in the form. Protip: Search for your ticket's name, which is the value you entered for "Provide a short description of what you were doing.*"

# Checklist:
